src/lang/po/Prusa-Firmware-Buddy.pot

```diff
@@ -305,264 +267,255 @@ msgstr ""
 msgid "LAN SETTINGS"
 msgstr ""
 
-#: src/gui/screen_menu_languages.cpp:119
+#: src/gui/screen_menu_languages.cpp:120
 msgid "LANGUAGES"
 msgstr ""
 
-#: src/gui/wizard/wizard_load_unload.cpp:44
+#. Ignore
+#: src/gui/dialogs/dialog_response.cpp:17
 msgid "LOAD"
 msgstr ""
 
-#: src/gui/dialogs/DialogFactory.cpp:12 src/gui/screen_menu_filament.cpp:45
+#: src/gui/dialogs/DialogFactory.cpp:13 src/gui/screen_menu_filament.cpp:47
 msgid "LOAD FILAMENT"
 msgstr ""
 
-#: src/guiapi/include/MItem_menus.hpp:138
-msgid "Lan settings"
+#: src/guiapi/include/MItem_menus.hpp:129
+msgid "Lan Settings"
 msgstr ""
 
-#: src/guiapi/include/MItem_menus.hpp:158
+#: src/guiapi/include/MItem_menus.hpp:149
 msgid "Language"
 msgstr ""
 
-#: src/guiapi/include/MItem_print.hpp:48
+#: src/guiapi/include/MItem_tools.hpp:19
 msgid "Live Adjust Z"
 msgstr ""
 
-#: src/gui/screen_menu_filament.cpp:44
+#: src/gui/screen_menu_filament.cpp:46
 msgid "Load Filament"
 msgstr ""
 
-#: src/gui/screen_menu_lan_settings.cpp:240
-msgid "Load settings"
+#: src/gui/screen_menu_lan_settings.cpp:201
+msgid "Load Settings"
 msgstr ""
 
-#: src/gui/screen_splash.cpp:53
+#: src/gui/screen_splash.cpp:43
 msgid "Loading ..."
 msgstr ""
 
-#: src/gui/dialogs/DialogLoadUnload.cpp:51
+#: src/gui/dialogs/DialogLoadUnload.cpp:28
 msgid "Loading to nozzle"
 msgstr ""
 
-#: src/guiapi/include/MItem_tools.hpp:209
+#: src/guiapi/include/MItem_tools.hpp:274
 msgid "Loud"
 msgstr ""
 
-#: lib/Prusa-Error-Codes/12/errors_list.h:57
-#: lib/Prusa-Error-Codes/12/errors_list.h:61
+#: src/guiapi/include/MItem_tools.hpp:368
+msgid "M.I.N.D.A."
+msgstr ""
+
+#. r=1, c=19
+#: lib/Prusa-Error-Codes/12_MINI/errors_list.h:66
+#: lib/Prusa-Error-Codes/12_MINI/errors_list.h:72
 msgid "MAXTEMP ERROR"
 msgstr ""
 
-#: src/gui/test/screen_mesh_bed_lv.cpp:89
-msgid "MESH BED L."
+#: src/gui/test/screen_mesh_bed_lv.cpp:56
+msgid "MESH BED LEVELING"
 msgstr ""
 
-#. p_window_header_set_icon(&(pmsg->header), IDR_PNG_status_icon_menu);					ICONka od Michala Fanty
-#: src/gui/screen_messages.cpp:61
+#. Rect16(10, 28, 11 * 20, 18 * 16))
+#: src/gui/screen_messages.cpp:21
 msgid "MESSAGES"
 msgstr ""
 
-#: lib/Prusa-Error-Codes/12/errors_list.h:65
-#: lib/Prusa-Error-Codes/12/errors_list.h:69
+#. r=1, c=19
+#: lib/Prusa-Error-Codes/12_MINI/errors_list.h:78
+#: lib/Prusa-Error-Codes/12_MINI/errors_list.h:84
 msgid "MINTEMP ERROR"
 msgstr ""
 
 #: src/gui/screen_menu_move.cpp:43
 msgid "MOVE AXIS"
 msgstr ""
 
-#: src/gui/dialogs/DialogLoadUnload.cpp:47
-msgid ""
-"Make sure the     \n"
-"filament is       \n"
-"inserted through  \n"
-"the sensor.       "
+#: src/gui/dialogs/DialogLoadUnload.cpp:24
+msgid "Make sure the filament is inserted through the sensor."
 msgstr ""
 
-#: src/gui/screen_print_preview.cpp:147 src/gui/screen_print_preview.cpp:155
+#: src/gui/screen_print_preview.cpp:128
 msgid "Material"
 msgstr ""
 
-#: src/gui/wizard/xyzcalib.cpp:78
+#: src/gui/wizard/xyzcalib.cpp:196
 msgid ""
 "Measuring reference\n"
```

Comment:
```suggestion
"Measuring reference height of calibration points"
```

Reply:
Have you seen the actual code? It is #if0 - will be fixed later. I'm not touching this.